test/integration/go.mod

```diff
@@ -1,10 +1,11 @@
-module github.com/terraform-google-modules/enterprise-application/test/integration
+module bootstrap
 
-go 1.21
+go 1.21.1
 
 require (
 	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.13.0
 	github.com/stretchr/testify v1.9.0
+	github.com/terraform-google-modules/enterprise-application/test/integration/testutils v0.0.0-00010101000000-000000000000
```

Comment:
This version number is correct?

Reply:
I suspect this was an artifact from before `testutils/transforms.go` was added to this PR.  Probably can revert the `go.*` files or run `go mod tidy`.